trojsten/results/templates/trojsten/results/parts/results_table.html

```diff
@@ -74,7 +74,10 @@
             {{ row.user.school.name }}
         </abbr>
     {% else %}
-        {{ row.user.school.name }}
+        <abbr title="Iná škola" data-toggle="tooltip"
+              data-placement="top">
+            ?
```

Comment:
Chceme tu zobrazovať otáznik? Podľa mňa cheme tiež iná škola. A v title chýba preklad.

Reply:
Preklady v htmlkach som nerobil lebo a) neviem ako b) pride mi to ako nieco co bude lepsie spravit hromadne v ramci samostatneho PR

Reply:
je to jednoduché: https://docs.djangoproject.com/en/2.0/topics/i18n/translation/#internationalization-in-template-code